modules/seqexec/web/client/src/main/resources/theme/semantic.less

```diff
@@ -0,0 +1,156 @@
+/*
+
+███████╗███████╗███╗   ███╗ █████╗ ███╗   ██╗████████╗██╗ ██████╗    ██╗   ██╗██╗
+██╔════╝██╔════╝████╗ ████║██╔══██╗████╗  ██║╚══██╔══╝██║██╔════╝    ██║   ██║██║
+███████╗█████╗  ██╔████╔██║███████║██╔██╗ ██║   ██║   ██║██║         ██║   ██║██║
+╚════██║██╔══╝  ██║╚██╔╝██║██╔══██║██║╚██╗██║   ██║   ██║██║         ██║   ██║██║
+███████║███████╗██║ ╚═╝ ██║██║  ██║██║ ╚████║   ██║   ██║╚██████╗    ╚██████╔╝██║
+╚══════╝╚══════╝╚═╝     ╚═╝╚═╝  ╚═╝╚═╝  ╚═══╝   ╚═╝   ╚═╝ ╚═════╝     ╚═════╝ ╚═╝
+
+  Import this file into your LESS project to use Semantic UI without build tools
+*/
+
+/* Global */
+& {
+  @import "~semantic-ui-less/definitions/globals/reset";
+}
+& {
+  @import "~semantic-ui-less/definitions/globals/site";
+}
+
+/* Elements */
+& {
+  @import "~semantic-ui-less/definitions/elements/button";
+}
+& {
+  @import "~semantic-ui-less/definitions/elements/container";
+}
+& {
+  @import "~semantic-ui-less/definitions/elements/divider";
+}
+// & {
```

Comment:
Commented widgets aren't used but i'll keep them here in case we need them later on